chaoscenter/graphql/server/pkg/authorization/tests/fuzz_test.go

```diff
@@ -0,0 +1,130 @@
+package tests
```

Comment:
Is there a reason to create a test dir? IMO, It would be better to locate the Unit test and Fuzzing test cases in the same filename_test.go and we don't have to make test dir every single pkg. WDYT?

Reply:
@namkyu1999 since we are integrating with OSS fuzz, we are currently working on creating a build script which will  pick up all the files with suffix *_test_fuzz.go under test directory. This is something we saw other projects are doing for better readability. Let me know your thoughts. Else we can just separate out the fuzz_tests from UTs.

Reply:
@Saranya-jena , Other CNCF projects use [this repository](https://github.com/cncf/cncf-fuzzing).

for example, `argo` project's[ build.sh](https://github.com/google/oss-fuzz/blob/master/projects/argo/build.sh) just calling `$SRC/cncf-fuzzing/projects/argo/build.sh`

And in the cncf-fuzzing repo, they write[ fuzz test code there](https://github.com/cncf/cncf-fuzzing/tree/main/projects/argo). How about utilizing them?

Reply:
But I think, that's because they are using CNCF fuzzing and we are using OSS-Fuzz right? 🤔 
We are taking reference from [CrossPlane](https://github.com/crossplane/crossplane/blob/master/test/fuzz/oss_fuzz_build.sh)

Reply:
https://github.com/google/oss-fuzz/pull/11608

Reply:
@Saranya-jena I checked, and the cncf-fuzz repo is simply a collection of fuzz-related code. Since that repo is not actively used at the moment, I think it would be better to go with your suggestion.

Reply:
but IMO, `tests` dir is unnecessary. how about not using `tests` dir?

FYI: https://github.com/crossplane/crossplane/tree/master/internal/dag

Reply:
Yes, will add it back to handler, that will be better. Can you check the implementation here: https://github.com/litmuschaos/litmus/pull/4483
If it's fine, I can make similar changes here.

Reply:
sure 🚀